pkg/storage/database/gorm/gorm.go

```diff
@@ -139,49 +139,50 @@ func (s *Gorm) getTeamId(userId uint) uint {
 	return uint(user.Teams[0].ID)
 }
 
-// AddAPIKey implements database.ProprietaryDB.
 func (*Gorm) AddAPIKey(ctx context.Context, destId int64, hashedAPIKey string) error {
 	panic("unimplemented")
 }
 
-// CreateDestination implements database.ProprietaryDB.
-func (s *Gorm) CreateDestination(ctx context.Context, userId uint, destType string, settings map[string]any) (config.Destination, error) {
+func (s *Gorm) CreateDestination(
+	ctx context.Context,
+	userId uint,
```

Comment:
This should really be a `teamId`. I feel like the userId -> teamId translation should happen before this function is called.